tests/integration.rs

```diff
@@ -4712,7 +4713,43 @@ mod tests {
                         .unwrap_or_default(),
                     1
                 );
+                // And check the block device can be read.
+                assert!(guest
+                    .ssh_command("dd if=/dev/vdc of=/dev/null bs=1M iflag=direct count=16")
+                    .is_ok());
+
+                // Let's remove it the extra disk.
+                assert!(remote_command(&api_socket, "remove-device", Some("test0")));
+                thread::sleep(std::time::Duration::new(5, 0));
+
```

Comment:
Would it be good to check that vdc is not present here?

Reply:
Oh yes good point!

Reply:
Done!